assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -163,6 +155,25 @@ che.workspace.startup_debug_log_limit_bytes=10485760
 # This configuration is mainly required for workspace idling when the OpenShift OAuth is enabled.
 che.workspace.stop.role.enabled=true
 
+
+# Name of cofig map in Che server namespace with additional CA TLS certificates to be propogated into all user's workspaces.
+# If the property is set on OpenShift 4 infrastructure, then cluster CA bundle will be propogated too.
```

Comment:
> then cluster CA bundle will be propogated too

that really depends on https://github.com/eclipse/che/pull/18377/files#diff-deca31b9cca47d5b0356dd75fc0fd0c8cb980f3aa029da27b475bdbb37353eb0R519
If admin will need to configure labels for dest-configmap, and override inject-ca, it won't be true. But this statement makes an impression at least for me, that it's handled by Che Server itself, independent from which labels are configured.

Reply:
I'll clarify the comment